### PR TITLE
add test to make sure that it is possible to not have any association…

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
…s serialized

related: https://github.com/rails-api/active_model_serializers/issues/1025#issuecomment-145054908

### CODE REVIEW

File: test/action_controller/json/include_test.rb [hunk 1]
```diff
@@ -36,6 +36,11 @@ def render_without_include
             render json: @author, adapter: :json
           end
 
+          def render_without_associations
+            setup_data
+            render json: @author, adapter: :json, include: []
```
Comment (beauby) [inline test/action_controller/json/include_test.rb]: There's no need anymore to go through a `render` call to test that kind of thing. Just pass the option to `SerializableResource`.

Comment (NullVoxPopuli) [inline test/action_controller/json/include_test.rb]: do we have a file that tests include output, specifically? or is it kinda scattered?

Comment (beauby) [inline test/action_controller/json/include_test.rb]: It is kinda adapter-specific, so no, not really.

Comment (bf4) [inline test/action_controller/json/include_test.rb]: unless you're testing the controller, unit test is better :)  also, see test/support, there's some serialization helpers in there

Comment (bf4) [inline test/action_controller/json/include_test.rb]: I want to remove most of the controller tests are that really now 'SerializableResource' tests.. would clean things up I think

Comment (NullVoxPopuli) [inline test/action_controller/json/include_test.rb]: :+1:

**Comment by NullVoxPopuli** [comment]
I moved the test. :-)

File: test/adapter/json/include_test.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    module Adapter
+      class Json
+        class IncludeTest < Minitest::Test
+          def setup
+            ActionController::Base.cache_store.clear
```
Comment (beauby) [inline test/adapter/json/include_test.rb]: This is not needed as the cache is automatically cleared before each test.

File: test/adapter/json/include_test.rb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+            @blog = Blog.new(id: 1, name: 'My Blog!!')
+            @post.blog = @blog
+            @author.posts = [@post]
+
+            @first_comment.author = @author
+            @second_comment.author = @author
+            @author.comments = [@first_comment, @second_comment]
+            @author.roles = []
+            @author.bio = {}
```
Comment (beauby) [inline test/adapter/json/include_test.rb]: Most of this is not needed. Let's try to keep the tests as minimal as possible. For now, only one association is needed for your test.

**Comment by NullVoxPopuli** [comment]
rebased

**Comment by NullVoxPopuli** [comment]
@beauby @bf4 ?

**Comment by NullVoxPopuli** [comment]
I'm pretty sure there are already tests that achieve what this test is doing.

**Pull request closed without merging**